### PR TITLE
Blind fix for exclude warnings for SPM

### PR DESCRIPTION
**Pull request by vanessa-hatch**
Removes the excluded folders from the SPM package manifest.

### CODE REVIEW

**Comment by vanessa-hatch** [comment]
FYI this hasn't been tested yet, I was also putting it up to see what kind of permissions I had on this repo. 😄

**Review by scottmarchant — APPROVED** [review]

**Comment by scottmarchant** [comment]
@downie-hatch Once you merge this, you'll need to tag a release version for it, then update the version consumed in our repo, then these warnings should go away.

Thanks for doing this!  There is a ticket for this, btw.

File: Package.swift [hunk 1]
```diff
@@ -23,11 +23,9 @@ let package = Package(
         // Targets can depend on other targets in this package, and on products in packages which this package depends on.
         .target(
             name: "EasyTipView",
-            dependencies: [],
-            exclude: ["assets", "Example", "Source"]),
```
Comment (scottmarchant) [inline Package.swift:27]: Oh, if these folders exist, we might need to look deeper

**Review by scottmarchant — COMMENTED** [review]
Sounds like the folders exist, we'll need to look deeper.

File: Package.swift (hunk 1 above)
Comment (scottmarchant) [reply Package.swift:27]: Nevermind.  Looks like your changes are correct.

**Review by josh-berlin — APPROVED** [review]

**Comment by zakkhoyt** [comment]
@downie-hatch @scottmarchant 

Our repo is currently pointing to a commit, not a tag. We could either use the new commit from this MR or point to `master`. Not sure that using tags for this private fork is worth it. 

zakkbookmax2022-05-26_11 02 31@2x

**Review by zakkhoyt — APPROVED** [review]
Changes look good assuming that the source branch was `30c95b4132bab7ac2a1c16c941eb52d0683c5baa`  (head of `master` which it looks like it was)

**Comment by vanessa-hatch** [comment]
oh my I definitely did not expect this to live this long. And now there's a conflict with Package.swift

I'm not up to speed enough about what this change was (even though I did it) to feel good about resolving conflicts and merging. Should we push forward with this or just start updating the Package.swift fresh?

**Comment by zakkhoyt** [comment]
I think this jist of this PR has already been merged into `master`. I bet if you pull the lastest into this MR that it will be equivalent. `master` has the exclude file list commented out (not fully removed). But same thing. 

https://github.com/hatch-baby/EasyTipView/blob/master/Package.swift

I think this PR can be closed.

**Comment by zakkhoyt** [comment]
Closing this PR. This fix already exists in `master`. This PR is also 9 months old.

**Pull request closed without merging**